### PR TITLE
feat(frontend): disable Carousel slide animation on init

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

The idea is to disable Carousel slide animation on init (on back/load). Also, I added `out` animation to dapps carousel containers, similar as we had them for `in`.

### CODE REVIEW

File: src/frontend/src/routes/(app)/+layout.svelte [hunk 1]
```diff
@@ -45,7 +45,7 @@
 		<SplitPane>
 			<NavigationMenu slot="menu">
 				{#if route === 'tokens'}
-					<div in:fade class="hidden xl:block">
+					<div in:fade out:fade class="hidden xl:block">
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/routes/(app)/+layout.svelte:48]: I think you can summarize `in:` and `out:` as `transition:` (https://svelte.dev/docs/svelte/in-and-out)

File: src/frontend/src/routes/(app)/+page.svelte [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	import Tokens from '$lib/components/tokens/Tokens.svelte';
 </script>
 
-<div in:fade class="mb-6 flex justify-center xl:hidden">
+<div in:fade out:fade class="mb-6 flex justify-center xl:hidden">
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/routes/(app)/+page.svelte:7]: ditto

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM! Tks

File: src/frontend/src/routes/(app)/+layout.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/routes/(app)/+layout.svelte:48]: Nice one!

**Pull request merged**